src/main/java/com/mercateo/rest/hateoas/client/impl/ResponseBuilder.java

```diff
@@ -27,10 +28,11 @@
 	private Client client;
 	@NonNull
 	private ObjectMapper objectMapper;
+	URI uri = URI.create("http://localhost:8080/");
```

Comment:
actually, the uri is no state of the responsebuilder, it should be a parameter of the the "build" - methods.
also i don't know, what the local host adress means here :-)